lib/sync/installation.js

```diff
@@ -102,14 +102,19 @@ module.exports.processInstallation = (app, queues) => {
       return processor(github, repository, cursor, perPage)
     }
 
+    const processJobGitHubError = (err) => {
+      if (!String(err).includes('MAX_NODE_LIMIT_EXCEEDED') ||
+          !String(err).includes('NOT_FOUND')) {
+        throw (err)
+      }
+    }
+
     const execute = async () => {
       for (const perPage of [20, 10, 5, 1]) {
         try {
           return await pagedProcessor(perPage)
         } catch (err) {
-          if (!String(err).includes('MAX_NODE_LIMIT_EXCEEDED')) {
-            throw err
-          }
+          processJobGitHubError(err)
```

Comment:
```suggestion
          handleGitHubError(err)
```

Two reasons:

1. This function is defined and only used within the context of a job so `Job` seems superfluous to me.
2. `handle` feels closer to what we're doing here: take an error, decide how handle it (throw or ignore). I use _feel_ intentionally. This is a matter of opinion and if you like `process` better, please do keep it! 😄